src/lib.rs

```diff
@@ -220,6 +224,12 @@ impl Window {
     ///  .expect("Unable to open Window");
     /// ```
     pub fn new(name: &str, width: usize, height: usize, opts: WindowOptions) -> Result<Window> {
+        if opts.transparency {
+            assert!(
```

Comment:
@emoon Should we assert here or return an Err value?

Reply:
Return an error code. `assert!` will only be present in debug config.

Reply:
Ok Im done here